src/main/java/cart/admin/controller/AdminRestController.java

```diff
@@ -0,0 +1,46 @@
+package cart.admin.controller;
+
+import cart.catalog.dto.ProductRequestDTO;
+import cart.catalog.dto.ProductResponseDTO;
+import cart.catalog.service.CatalogService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/admin")
+public class AdminRestController {
+
+    private final CatalogService catalogService;
+
+    public AdminRestController(final CatalogService catalogService) {
+        this.catalogService = catalogService;
+    }
+
+    @GetMapping("/products")
+    public ResponseEntity<List<ProductResponseDTO>> display() {
+        final List<ProductResponseDTO> items = this.catalogService.display();
+        return ResponseEntity.status(HttpStatus.OK).body(items);
```

Comment:
```suggestion
        return ResponseEntity.ok(items);
```
좀 더 간단하게 표현할 수도 있겠네요!